src/qt/walletview.cpp

```diff
@@ -235,7 +242,9 @@ void WalletView::setClientModel(ClientModel *clientModel)
     sendZcoinView->setClientModel(clientModel);
     znodeListPage->setClientModel(clientModel);
     exoAssetsPage->setClientModel(clientModel);
-    sigmaView->setClientModel(clientModel);
+    if (pwalletMain->IsHDSeedAvailable()) {
```

Comment:
We should check if `sigmaView` is not null instead so the logic to control the visibility of `sigmaView` will be in the single place.